ci/release-info-v3.py

```diff
@@ -0,0 +1,100 @@
+import json
+import os
+import re
+import sys
+from datetime import date
+from subprocess import Popen, PIPE
+
+def check_version(ver):
+    # Checks package version for matching with the current tag reference
+    # print(ver[0])
+    if ref is not None and ref != 'refs/tags/v' + str(ver[0]):
+        return 0
+    else:
+        return 1
+
+def print_error(crate, ver, v):
+    # print(crate, ver[0], v)
+    if not v:
+        print(
+            '::error file={path}::version {0} does not match release tag {1}'
+            .format(str(ver[0]), ref, path = str(crate) + '/Cargo.toml')
+        )
+
+def bundle_version(crates):
+    # Reads package versions from workspace manifest file
+    channel = Popen(
+        ['cargo', 'metadata', '--format-version=1', '--no-deps'],
+        stdout=PIPE
+    )
+
+    # parse json data
+    data = json.load(channel.stdout).get('packages')
+
+    # read, map and assign workspace crates versions to bundle package versions
+    for package_id, _ in enumerate(data):
+        if data[package_id]['name'] in crates:
+            crates[data[package_id]['name']].append(data[package_id]['version'])
+    #       print(package_id, data[package_id]['name'], data[package_id]['version'], crates)
```

Comment:
Remove any temp lines that were commented out.

Reply:
OK, yeah, this was for testing and control of the correct output for this preliminary PR. Thanks for mention that! Will remove for sure!